google/cloud/spanner/internal/session_pool.cc

```diff
@@ -297,11 +297,25 @@ Status SessionPool::CreateSessions(
 }
 
 StatusOr<SessionHolder> SessionPool::Allocate(bool dissociate_from_pool) {
```

Comment:
observation: the order of definitions in here do not match the order in the header. It predates this PR.

Reply:
Ack, but I'll leave it for now.